sds_data_manager/lambda_code/SDSCode/upload_api.py

```diff
@@ -67,11 +67,11 @@ def lambda_handler(event, context):
             ),
         }
     # TODO: Handle other filetypes other than science files
-    #      The current ScienceFilepathManager only accepts filenames, not the full path
+    #      The current ScienceFilePath only accepts filenames, not the full path
     filename = os.path.basename(path_params)
```

Comment:
Would it be useful to let ScienceFilePath take a filename or a full path? That might be worth adding at some point, since the imap-data-access CLI uses the same pattern of accepting either a filename or a full path...

Reply:
yeah, I don't mind either way.